modules/machine-health-checks-pausing.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-restricted-network-cluster.adoc
+
+[id="machine-health-checks-pausing_{context}"]
+= Pausing a MachineHealthCheck resource
+
+During the upgrade process, nodes in the cluster might become temporarily unavailable. In case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+1. To list all the available `MachineHealthCheck` resources that you want to pause, run the following command:
++
+[source,terminal]
+----
+$ oc get machinehealthcheck -n openshift-machine-api
+----
+
+2. To pause the machine health checks, add the `cluster.x-k8s.io/paused=""` annotation to the `MachineHealthCheck` resource. Run the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api annotate mhc <mhc-name> cluster.x-k8s.io/paused=""
+----
++
+The annotated machine health check resource resembles the following YAML file:
```

Comment:
I'd recommend either using the [general](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) reference as "machine health check" (without "resource") or the [actual value](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) as "`MachineHealthCheck` resource" to align with our guidelines.